js/containers/app.js

```diff
@@ -61,8 +62,9 @@ class App extends PureComponent {
 
   renderApp () {
     const { dataFetching, earthquakes, volcanoes, layers, crossSectionPoints, mapStatus, setMapStatus, updateEarthquakesData,
-      mark2DViewModified, mark3DViewModified, mode, setCrossSectionPoint, changedViews } = this.props
+      mark2DViewModified, mark3DViewModified, mode, setCrossSectionPoint, changedViews, pins, setPin } = this.props
     const showUI = config.showUserInterface
+    const pinUI = config.clickToMoveSinglePin
```

Comment:
Shouldn't we add a new option? 
It feels a bit strange to connect simplified UI with clickToMoveSinglePin. ;)